packages/web-components/.storybook/main.js

```diff
@@ -222,15 +222,17 @@ module.exports = {
           {
             loader: 'postcss-loader',
             options: {
-              plugins: () => {
-                const hostPseudo =
-                  require('../tools/postcss-fix-host-pseudo')();
-                const autoPrefixer = require('autoprefixer')({
-                  overrideBrowserslist: ['last 1 version', 'ie >= 11'],
-                });
-                return !useRtl
-                  ? [hostPseudo, autoPrefixer]
-                  : [rtlcss, hostPseudo, autoPrefixer];
+              postcssOptions: {
+                plugins: () => {
+                  const hostPseudo =
+                    require('../tools/postcss-fix-host-pseudo')();
+                  const autoPrefixer = require('autoprefixer')({
+                    overrideBrowserslist: ['last 1 version', 'ie >= 11'],
+                  });
+                  return !useRtl
+                    ? [hostPseudo, autoPrefixer]
+                    : [rtlcss, hostPseudo, autoPrefixer];
```

Comment:
Just curious, is there a good way to check that we're actually getting `rtlcss` post css processing?

Reply:
yeah there's [rtlcss directives ](https://rtlcss.com/learn/getting-started/why-rtlcss/#Processing-Directives) that we used to have...but I don't see them anymore - let me check

Reply:
I'll merge it in for now to unblock this issue, but this would require more investigation to see why rtlcss styles aren't coming through in storybook